### DISCUSSION TITLE
Real-ESRGAN Anime Model

### DISCUSSION THREAD

**Discussion by KAGEYAM4** [upscayl/upscayl#1025] [Q&A] [answered]
### Checklist

- [X] I have checked that this issue isn't addressed in [the FAQ](https://github.com/upscayl/upscayl#-faq).
- [X] I have searched for this in the [issues tab](https://github.com/upscayl/upscayl/issues?q=).

### Description

After looking into custom model, it seems that Real-ESRGAN already provides ncnn. Would it be possible to add this model natively in upscayl?

But i am also confused of which of there two links model should be used -->
1. https://github.com/xinntao/Real-ESRGAN/blob/master/docs/anime_model.md#how-to-use
2. https://github.com/xinntao/Real-ESRGAN-ncnn-vulkan

I tried https://github.com/xinntao/Real-ESRGAN/blob/master/docs/anime_model.md#ncnn-executable-file, downloaded the ubuntu version and improted into upscayl but upscayl read all the model except realesrgan-x4plus-anime. So i had to instead use cli provided by the zip file.

**Comment by aaronliu0130** [comment, marked as answer]
This is included as the digital art model.

**Reply by KAGEYAM4** [reply]
I had no idea that they were same. Is this documented, like model source. Maybe In the app, tool-tip can be added showing the creator.

**Reply by KAGEYAM4** [reply]
@aaronliu0130  similar to how other model sources are written in brackets. Can `Digital Art` original source be also listed in brackets? 
![image](https://github.com/user-attachments/assets/8ec0e213-0c7e-4975-952d-c6b69fc0806f)

And may an `info` icon linking to original model page
